parking-lot-management/src/main/java/lld/parkinglotmanagement/entity/Ticket.java

```diff
@@ -0,0 +1,47 @@
+package lld.parkinglotmanagement.entity;
+
+import java.time.LocalDateTime;
+
+public class Ticket {
+
+    private final String ticketNo;
```

Comment:
can we change type to Integer instead String ? it may help in heap operations ?
Or shall we introduce new attribute "sequence'  ?